### DISCUSSION TITLE
Parallel trading mode in latest beta release

### DISCUSSION THREAD

**Discussion by himanshu8garg** [gnzsnz/ib-gateway-docker#68] [Q&A] [answered]
This is the first version with support for TRADING_MODE=both which allows to start in parallel live and paper instances. Has anyone had any luck with this. I tried this and I am getting ```.> Invalid TRADING_MODE: both```

**Comment by gnzsnz** [comment]
Make sure you are using 10.27.1b

**Reply by gnzsnz** [reply]
for tws-rdesktop

`docker pull ghcr.io/gnzsnz/tws-rdesktop:10.27.1b`

for ib-gateway

`docker pull ghcr.io/gnzsnz/ib-gateway:10.27.1b`

![image](https://github.com/gnzsnz/ib-gateway-docker/assets/8376642/15d86201-05c6-40e6-94b4-ed2745930057)

**Reply by himanshu8garg** [reply, marked as answer]
Hey, thanks for the reply I was building the image myself, maybe I missed upgrading to 10.27.1b somewhere. Anyways pulled the image directly and it works.
